spec/models/user_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 RSpec.describe User do
   it 'validates the name is unique' do
-    create(:user)
-    expect(build(:user)).to be_invalid
+    create(:user, name: 'joe')
+    expect(build(:user, name: 'joe')).to be_invalid
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.